File: src/main/java/level_a/java/SerejaAndDima.java
package level_a.java;

import java.io.*;

public class SerejaAndDima {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        br.readLine();
        String[] cards = br.readLine().split("\\s+");
        int[] players = new int[2];
        int i=0;
        int j=cards.length-1;
        for(int k=0; k<cards.length; k++){
            if(Integer.parseInt(cards[i])> Integer.parseInt(cards[j])){
                players[k % 2] += Integer.parseInt(cards[i]);
                i++;
            }else{
                players[k % 2] += Integer.parseInt(cards[j]);
                j--;
            }
        }
        System.out.printf("%d %d", players[0], players[1]);
    }
}

File: src/main/java/level_a/kotlin/SerejaAndDima.kt
package level_a.kotlin

import java.io.*

fun main(args: Array<String>){
    val br = BufferedReader(InputStreamReader(System.`in`))
    br.readLine()
    val cards = br.readLine().split("\\s+".toRegex())
    val players = arrayOf(0,0)
    var i = 0;
    var j = cards.size-1
    for(k in cards.indices){
        players[k%2]+= if (cards[i].toInt()>cards[j].toInt()){
            cards[i++].toInt()
        }else{
            cards[j--].toInt()
        }
    }
    print("${players[0]} ${players[1]}")
}